deepmd/train/run_options.py

```diff
@@ -202,15 +141,9 @@ def __init__(
         restart: Optional[str] = None,
         log_path: Optional[str] = None,
         log_level: int = 0,
-        mpi_log: str = "master",
-        try_distrib: bool = False
```

Comment:
Please fix tests as you removed this argument.

Reply:
Thanks. New commit is pushed.